app/retail/views.py

```diff
@@ -26,7 +26,14 @@
 
 
 def index(request):
+    slides = [
```

Comment:
i love that this is abstracted out of the template :)

Reply:
credits go to @tra38  :D